.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: dart
+
+dart:
+  - dev
+  - stable
+
+dart_task:
+  - test
+  - dartfmt
+  - dartanalyzer
```

Comment:
We shouldn't run the formatter and analyzer on both dev and stable; see [the project template](https://github.com/dart-lang/oss_project_templates/blob/master/dart-github-dart-lang/.travis.yml).

Reply:
ack